### PR TITLE
feature(extension): tell user how to find Accessibility Checker view if in Accessibility Assessment view first time after install only.

### PR DESCRIPTION
**Pull request by drjoho**
* New Rule(s): **List rule IDs**
* New or modified help files: **List rule IDs**
* Rule bug: **List rule IDs**
* Extension UI bug
* Automated tool bug
* Other (Provide information)

### This PR is related to the following issue(s): 
- 

### Additional information can be found here: 
- 

### Testing reference: 
- 

### I have conducted the following for this PR: 
- [x] I validated this code in Chrome and FF 
- [x] I validated this fix in my local env
- [x] I provided details for testing
- [x] This PR has been reviewed and is ready for test  
- [ ] I understand that the title of this PR will be used for the next release notes.

### CODE REVIEW

**Comment by drjoho** [comment]
There is one more issue to fix as it should only put up the panel or modal if in Accessibility Assessment view.

File: accessibility-checker-extension/src/assets/img/Chrome.svg
Comment (tombrunet) [inline accessibility-checker-extension/src/assets/img/Chrome.svg:1]: Screenshots are showing the wrong thing. Should be on the Elements / Styles subpanel and then show clicking the subpanel dropdown to get to the "Accessibility Checker" (not the "Accessibility Assessment")

File: accessibility-checker-extension/src/assets/img/Edge.svg
Comment (tombrunet) [inline accessibility-checker-extension/src/assets/img/Edge.svg:1]: Same problem as Chrome screenshot

File: accessibility-checker-extension/src/assets/img/Firefox.svg
Comment (tombrunet) [inline accessibility-checker-extension/src/assets/img/Firefox.svg:1]: Same problem as Chrome screenshot

File: accessibility-checker-extension/src/ts/background/backgroundController.ts [hunk 1]
```diff
@@ -455,7 +455,7 @@ class BackgroundController extends Controller {
         if (!("tabStopLines" in settings)) { (settings as ISettings).tabStopLines = true; }
         if (!("tabStopOutlines" in settings)) { (settings as ISettings).tabStopOutlines = false; }
         if (!("tabStopAlerts" in settings)) { (settings as ISettings).tabStopAlerts = true; }
-        if (!("tabStopFirstTime" in settings)) { (settings as ISettings).tabStopFirstTime = true; }
+        if (!("checkerViewAwareFirstTime" in settings)) { (settings as ISettings).checkerViewAwareFirstTime = true; }
```
Comment (tombrunet) [inline accessibility-checker-extension/src/ts/background/backgroundController.ts:458]:
```suggestion
        if (!("tabStopFirstTime" in settings)) { (settings as ISettings).tabStopFirstTime = true; }
        if (!("checkerViewAwareFirstTime" in settings)) { (settings as ISettings).checkerViewAwareFirstTime = true; }
```

File: accessibility-checker-extension/src/ts/devtools/components/checkerViewAware.tsx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        BrowserDetection.setDarkLight();
+        let usrAgent = navigator.userAgent;
+        let elementTabImg: string | undefined;
+        if (usrAgent.indexOf("Chrome") > -1) {
+            elementTabImg = chromeImg;
+        } else if (usrAgent.indexOf("Firefox") > -1) {
+            elementTabImg = firefoxImg;
+        } else if (usrAgent.indexOf("Edge") > -1) {
+            elementTabImg = edgeImg;
+        }
```
Comment (tombrunet) [inline accessibility-checker-extension/src/ts/devtools/components/checkerViewAware.tsx:63]:
```suggestion
        if (usrAgent.include("Edg/") {
            elementTabImg = edgeImg;
        } else if (usrAgent.includes("Firefox/")) {
            elementTabImg = firefoxImg;
        } else {
            elementTabImg = chromeImg;
        }
```

File: accessibility-checker-extension/src/ts/devtools/components/checkerViewAware.tsx [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+            <Grid>
+                <Column sm={{ span: 4 }} md={{ span: 8 }} lg={{ span: 8 }}>
+                    <div className="modalContent">
+                        <div style={{marginBottom:"1rem"}}>
+                            You are in the <b>Accessibility Assessment</b> panel of the Checker.
+                        </div>
+                        <div style={{marginBottom:"1rem"}}>
+                            For code scanning functionality, navigate to the Accessibility Checker 
+                            panel within the Elements panel in the Dev Tools. Be aware that this 
+                            +panel can be nested within the dev tools navigation bar.
```
Comment (tombrunet) [inline accessibility-checker-extension/src/ts/devtools/components/checkerViewAware.tsx:81]:
```suggestion
                            panel can be nested within the dev tools navigation bar.
```

File: accessibility-checker-extension/src/ts/devtools/components/checkerViewAware.tsx [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+                        </div>
+                        <div style={{marginBottom:"1rem"}}>
+                            <img src ={elementTabImg} alt="Checker view on Elements Tab"></img>
```
Comment (tombrunet) [inline accessibility-checker-extension/src/ts/devtools/components/checkerViewAware.tsx:84]:
```suggestion
                            <img src={elementTabImg} alt="Checker view on Elements Tab"></img>
```

**Review by tombrunet — CHANGES_REQUESTED** [review]

File: accessibility-checker-extension/src/assets/img/Edge.svg
Comment (drjoho) [reply accessibility-checker-extension/src/assets/img/Edge.svg:1]: Done new screenshot.

**Review by tombrunet — APPROVED** [review]

**Pull request merged**